pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -629,5 +618,15 @@ pub mod pallet {
 				} => Self::process_outbound_message(sender, destination, message),
 			}
 		}
+
+		/// Process a message.
+		fn max_processing_weight(msg: &Self::Message) -> Weight {
+			match msg {
+				GatewayMessage::Inbound { message, .. } => {
+					LP_DEFENSIVE_WEIGHT.saturating_mul(message.unpack().len() as u64)
+				}
+				GatewayMessage::Outbound { .. } => LP_DEFENSIVE_WEIGHT, /* TODO can be much lower */
+			}
+		}
```

Comment:
This is the required weight the block should be to start processing the message. I think the outbound case can be much less, but not sure how many. We can leave it as it is because later, the real weights coming from the router will be used.